keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/specs/MaintainRolesSpecification.groovy

```diff
@@ -48,6 +49,7 @@ class MaintainRolesSpecification extends GebReportingSpec {
         maintainRolesLink.click()
 
         when: "i perform a search"
+        at UserSearchPage
```

Comment:
I think that just adding and `at` will perform a `waitFor` anyway so don't explicitly need them when we have `at`